keanu-python/tests/test_net.py

```diff
@@ -62,3 +63,32 @@ def test_probe_for_non_zero_probability_from_bayes_net() -> None:
 
     assert gamma.has_value()
     assert poisson.has_value()
+
+
+def test_can_save_and_load() -> None:
+    PROTO_FILE = "temp.proto"
```

Comment:
PyTest has a cheeky feature called `tmpdir` that you can use out the box. See `test_lambdas.py` for an example.